app/styles/modules/_settings.scss

```diff
@@ -573,3 +573,49 @@ section.modal-panel {
     margin-bottom: 30px;
   }
 }
+
+.fxa-radio {
```

Comment:
We can put the `type='radio'` styles into _base.scss https://github.com/mozilla/fxa-content-server/blob/c80d1aa9adba3829696f330566f36848d3001a8d/app/styles/_base.scss#L148